src/test/java/org/breedinginsight/brapps/importer/ExperimentFileImportTest.java

```diff
@@ -86,6 +86,8 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ExperimentFileImportTest extends BrAPITest {
     private static final String OVERWRITE = "overwrite";
+    private static final String APPEND_OVERWRITE_WORKFLOW = "append-dataset";
```

Comment:
When we merge the tests there are a couple things to check. In BI-2132 test setup the workflow id is grabbed from the workflows endpoint based on the expected order. It indirectly tests the ordering. There were some append tests that also had create new experiment calls, so we'll just want to make sure those are all good.

Reply:
I added an integer arg to ExperimentFileImportTest#getExperimentWorkflowId and use it for assigning both ids, now.